README.markdown

```diff
@@ -249,6 +251,8 @@ After the v0.8.6 release of node, nvm tries to install from binary packages. But
 
     nvm install -s 0.8.6
 
+If setting the `default` alias does not establish the node version in new shells (i.e. `nvm current` yields `system`), ensure that the system's node PATH is set before the `nvm.sh` source line in your shell profile (see [#658](/../../issues/658))
```

Comment:
will `/../../issues/658` link properly? it might be better to put the full URL, if github won't auto-link `#658`